package/test/package-pi.yaml

```diff
@@ -0,0 +1,178 @@
+apiVersion: packaging.carvel.dev/v1alpha1
+kind: PackageInstall
+metadata:
+  annotations:
+    kctrl.carvel.dev/local-fetch-0: .
+  creationTimestamp: null
+  name: cliplugin
+spec:
+  packageRef:
+    refName: cliplugin.cli.tanzu.vmware.com
+    versionSelection:
+      # XXX sufficient to pick up any version of package deployed during test
+      constraints: "<1.0.0"
+  serviceAccountName: tanzu-cliplugins-package-sa
+status:
+  conditions: null
+  friendlyDescription: ""
+  observedGeneration: 0
```

Comment:
`status` should be removed as we are using this to apply the `PackageInstall` on the cluster.